### PR TITLE
improved layout_product-cards

### PR DESCRIPTION
**Pull request by a-kovalov**
- [DEMO LINK](https://a-kovalov.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://a-kovalov.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/card.scss [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  &-group {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 19px;
+  }
+
+  &--position {
+    margin-left: 22px;
```
Comment (etojeDenys) [inline src/styles/blocks/card.scss:78]: why do you need this

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+.amount {
+  font-weight: 700;
+  font-size: 16px;
+  padding-top: 3px;
+  color: $color-main-accent;
+
+  &--position {
+    margin-left: 85px;
+  }
```
Comment (etojeDenys) [inline src/styles/blocks/card.scss:104]: i think it's redundant

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Comment by a-kovalov** [comment]
- [DEMO LINK](https://a-kovalov.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://a-kovalov.github.io/layout_product-cards/report/html_report/)

**Review by etojeDenys — APPROVED** [review]
lgtm

**Pull request closed without merging**